commit 99ee3f23319a8445584ea50e4dd44a0357232315 (refs/changes/04/4504/2)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2018-07-11T18:00:36+02:00 (1 year, 3 months ago)
    
    TRITON-582 Update sdcadm to latest progbar (1.2.0)
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/package.json b/package.json
index 8a05c11..f25ff4b 100644
--- a/package.json
+++ b/package.json
@@ -19,7 +19,7 @@
     "mkdirp": "0.3.5",
     "node-uuid": "1.4.1",
     "once": "1.3.0",
-    "progbar": "1.1.0",
+    "progbar": "1.2.0",
     "read": "1.0.5",
     "restify-clients": "1.4.0",
     "sdc-clients": "10.3.0",
@@ -39,7 +39,7 @@
     "eslint-plugin-joyent": "~2.0.0",
     "mock-fs": "4.4.x",
     "tap": "^12.0.1",
-    "zkstream": "0.10.1"
+    "zkstream": "0.11.2"
   },
   "engines": {
     "node": ">=0.10"
